content/documentation-v1/getting-started.md

```diff
@@ -5,146 +5,134 @@ weight: 6080
 
-#### `References.yml`
+#### 📄 references.yaml
 
 This file includes reference information used to create in-text citations and bibliographies that are designed to meet scholarly needs. Read more in [*Citations & Bibliographies*](/documentation/citation-bibliographies/).
 
-### 📁 static
+### 📁 _computed
+
+TK
+
+### 📁 _assets
+
+This directory includes anything that will be included in your final publication that doesn’t have to first be processed through Quire’s {{< q-def "static site generator" >}}, such as `fonts`, `images`, and `styles`. The `javascript` directory will primarily be used by developers. Please see [*Files For Developers*](/documentation/getting-started/#files-for-developers) for more information about how JavaScript is used in Quire.
 
-The `static` directory includes anything that will be included in your final publication, but that doesn’t have to first be processed through Quire’s {{< q-def "static site generator" >}}. By default, this includes a `css` directory for directly overriding theme styles (read more in [*Style Customization*](/documentation/styles-customization/)); a `downloads` directory for the multiple Quire formats ([*Output Your Project*](/documentation/multiformat-output/)); and an `img` directory for all image and other media assets ([*Figure Images*](/documentation/figure-images/)).
+#### 📁 fonts
 
-### 📄 README.md
+This directory is where the default Quire fonts are located including ["Noto Sans"](https://github.com/googlefonts/noto-fonts), ["Noto Serif"](https://github.com/googlefonts/noto-fonts), and ["IBM Plex Sans Condensed"](https://github.com/IBM/plex). Learn how to change and add fonts in the [Font Customization](/documentation/fonts-customization/) section of this guide.
 
-The `README.md` file is a code convention, and is a free space for information about the publication. **It is not used in the output Quire publication at all.** However, if you host your Quire project on {{< q-def "GitHub" >}} or other similar `git` project management sites, the `README.md` file is used for the repository’s front page description. Often it will include notes on development, on what usage is allowed, on how issues will be handled, and if contributions should be considered.
+#### 📁 images
+
+This directory is where publication images are stored. The `src` value in the `figures.yaml` file references the files names of the images as they are saved in this directory.
+
+
+#### 📁 styles
+
+This directory includes a `variables.scss` file and a `custom.css` file for directly overriding theme styles (read more in [*Style Customization*](/documentation/styles-customization/)).
 
 ## Files for Developers
 
-Inside each Quire project, you will find the following directories and files. Developers will primarily use the `config.yml` file and the `bin`, `config`, `site`, and `theme` directories.
+Developers will primarily use the `_includes`, `_layouts`, `_plugins` directories and the `config.yaml` file.
```

Comment:
I actually think `config.yaml` is more geared to the non-technical user. Developer-level configuration would happen other places.